scripts/build.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/bash
```

Comment:
I'd recommend using `!/usr/bin/env bash` here -- it's not always desirable to use plain `/bin/bash` over the current environment.